<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVariableWriteProtectDialog</class>
 <widget class="QDialog" name="CVariableWriteProtectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="enableWriteCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>可写</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>限值范围</string>
   </property>
   <widget class="QLineEdit" name="minValueEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxValueEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="expressButton1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="expressButton2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>超限处理</string>
   </property>
   <widget class="QRadioButton" name="proectTypeRadioButton0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>忽略</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="proectTypeRadioButton1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用限值</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="enableWriteProtectCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>写保护</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
